ipam/calico-ipam.go

```diff
@@ -14,24 +14,24 @@
 package main
 
 import (
+	"context"
 	"encoding/json"
 	"flag"
 	"fmt"
 	"net"
-
 	"os"
 
-	log "github.com/sirupsen/logrus"
-
 	"github.com/containernetworking/cni/pkg/skel"
-	"github.com/containernetworking/cni/pkg/types"
+	cniTypes "github.com/containernetworking/cni/pkg/types"
```

Comment:
it's unusual to use camelcase for imports isn't it, or am i making that?

Reply:
we had camelcase imports at a bunch of places, but you're right, it feels weird. I'll make it all lowercase

Reply:
yeah I'm not overly fussed since it's done.